core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -447,7 +447,15 @@ public String getServiceHost(String serviceName, Integer servicePort) {
      * @return a port that can be used for accessing the service container.
      */
     public Integer getServicePort(String serviceName, Integer servicePort) {
-        return ambassadorContainer.getMappedPort(ambassadorPortMappings.get(getServiceInstanceName(serviceName)).get(servicePort));
+        Map<Integer, Integer> portMap = ambassadorPortMappings.get(getServiceInstanceName(serviceName));
+
+        if (portMap == null) {
+            throw new IllegalArgumentException("Could not get a port for '" + serviceName + "'. " +
+                "Reason: '" + serviceName + "' does not exist. " +
```

Comment:
Sorry, one more tiny tweak. I'll commit this myself.
```suggestion
                "Testcontainers does not have an exposed port configured for '" + serviceName + "'. +
```

Reply:
No worries, sorry for late committing. I do not find the settings for this. Can you help me here?